core/domain/exp_services.py

```diff
@@ -267,7 +267,7 @@ def get_all_exploration_summaries():
 
 # Methods for exporting states and explorations to other formats.
 def export_to_zip_file(exploration_id, version=None):
-    """Returns a ZIP archive of the exploration.
+    """Returns a ZIP delete of the exploration.
```

Comment:
This replacement does not make sense, please check other files as well to ensure there aren't other similar cases.